### PR TITLE
Remove omarroth mentions

### LINKED ISSUE #1621: Invidious still contain mention of omarroth in a lot of places
Invidious, obviously, will always be a project that exist because of Omarroth. 

However, Invidious should be updated to reflect his departure.

I noticed *some* places where either the old Github URL appears, his nickname, or his email address (example: https://github.com/iv-org/invidious/blob/master/src/invidious/users.cr#L430 )

This should be updated to mention the new Github URL, "invidious" instead of his nickname, the Github URL instead of his email address.

### PR DESCRIPTION
**Pull request by hackerncoder**
Fix #1621.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# Note: This is no longer updated and links to omarroths repo, which doesn't exist anymore.
```
Comment (saltycrys) [inline CHANGELOG.md:1]: I though you meant removing email addresses in the changelog, updating repo links is fine.

**Review by saltycrys — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (hackerncoder) [reply CHANGELOG.md:1]: There are too many. Too many that link to the wiki which have to be rewritten to point to the documentation. I don't want to touch that right now.

**Comment by TheFrenchGhosty** [comment]
Is that everything? Or just what you found?

**Review by TheFrenchGhosty — APPROVED** [review]

**Comment by hackerncoder** [comment]
@TheFrenchGhosty What I found. But I did a grep recursive from the base of the repo for "omarroth" (and one just now that was also case insensitive) so it should be all.

**Comment by TheFrenchGhosty** [comment]
Alright, thanks!

**Pull request merged**